### PR TITLE
Changed loggly account and tweaked log size closes #3049

### LINKED ISSUE #3049: switch to new loggly account
We've got a ton of old stale data in our existing account. Let's start fresh.

### PR DESCRIPTION
**Pull request by myleshorton**

### CODE REVIEW

File: src/github.com/getlantern/flashlight/logging/logging.go [hunk 1]
```diff
@@ -57,9 +57,9 @@ func Init() error {
 	}
 	logFile = rotator.NewSizeRotator(filepath.Join(logdir, "lantern.log"))
 	// Set log files to 1 MB
-	logFile.RotationSize = 1 * 1024 * 1024
+	logFile.RotationSize = 4 * 1024 * 1024
 	// Keep up to 20 log files
-	logFile.MaxRotation = 20
+	logFile.MaxRotation = 5
```
Comment (atavism) [inline src/github.com/getlantern/flashlight/logging/logging.go]: Might just want to update the comments to reflect your changes.

Comment (myleshorton) [inline src/github.com/getlantern/flashlight/logging/logging.go]: Good point will do!

Comment (myleshorton) [inline src/github.com/getlantern/flashlight/logging/logging.go]: Done!

**Comment by atavism** [comment]
Ok, LGTM!

**Comment by myleshorton** [comment]
Awesome thanks @atavism!

**Pull request merged**